test/faker/default/test_lorem_pixel.rb

```diff
@@ -1,12 +1,36 @@
 # frozen_string_literal: true
 
 require_relative '../../test_helper'
+class FakeStdErr
+  attr_accessor :messages
+
+  def initialize
+    @messages = []
+  end
+
+  def write(msg)
+    @messages << msg
+  end
+end
 
 class TestLoremPixel < Test::Unit::TestCase
   def setup
     @tester = Faker::LoremPixel
   end
 
+  def test_deprecation_message
+    # rubocop:disable Style/GlobalVars
+    original_stderr = $stderr
+    fake_std_err    = FakeStdErr.new
+    $stderr         = fake_std_err
+    @tester.image
+    warn_message = 'NOTE: Faker::LoremPixel.image is deprecated; use Faker::LoremFlickr.image instead. It will be removed on or after 2022-12'
+    assert_includes(fake_std_err.messages[0], warn_message)
+  ensure
+    $std_err = original_stderr
+    # rubocop:enable Style/GlobalVars
+  end
+
```

Comment:
Thank you for adding a test for it, @uzorjchibuzor 

What do you think of keeping it simple? I feel like we could just have a simple assertion like such:

```suggestion
def test_image_deprecation_message
    out, err = capture_output do
      @tester.image(size: '3x3')
    end

    assert_match(/Faker::LoremPixel.image is deprecated; use Faker::LoremFlickr.image instead\./, err)
  end
```